File: Topic0102/006_herons_method/check_sqrt_integer.java
class Check_Sqrt_Integer{
  public static void main(String[] args){
    for(int i = 0; i < 20; i++)
      System.out.println("square root of " + i + " is integer : " + isXInteger(i));
  }

  static boolean isXInteger(int n){
    boolean y = false;

    for(int i = 1; i <= n; i++)
        if(i * i == n)
          y = true;

      return y;
    }
}

File: Basic_Algorithms/006_herons_method/check_sqrt_integer.js
var x = 2;

console.log(`square root of ${x} is integer : ${isXInteger(x)}.`);

function isXInteger(n) {
  y = false;

  for (let i = 1; i <= n; i++) if (i * i == n) y = true;

  return y;
}
